Composer/packages/client/src/pages/botProject/runtime-settings/RuntimeSettings.tsx

```diff
@@ -43,6 +45,14 @@ import {
 
 type RuntimeType = 'path' | 'command';
 
+/** Determine if a bot is configured to use the adaptive runtime vs the legacy runtime */
+export const isAdaptiveRuntime = (settings: DialogSetting): boolean => {
+  return (
+    settings?.runtime?.key === 'csharp-azurewebapp-v2' ||
+    (settings?.runtime?.key.match(/^adaptive-runtime/) ? true : false)
```

Comment:
I didn't do a fancy regex, I just used startsWith.  Let me know if you think the regex is necessary and I can update the function I wrote.

Reply:
nope that's fine, this regex is doing the same thing as startswith